internal/app/carrier/service.go

```diff
@@ -0,0 +1,24 @@
+package carrier
+
+import (
+	"context"
+	"parcel-service/internal/app/model"
+	svc "parcel-service/internal/app/service"
+	"time"
+)
+type service struct {
+	repo svc.CarrierAcceptRepository
+}
+
+
+// NewService is to generate for new repo
+func NewService(repo svc.CarrierAcceptRepository) *service {
+	return &service{
+		repo: repo,
+	}
+}
+
+// Assign a parcel with a carrier is to accept a carrier and other request will be rejected
+func (s *service) AssignCarrierToParcel(ctx context.Context, parcel model.CarrierRequest) error {
+	return s.repo.UpdateCarrierRequest(ctx, parcel, 2, 3, 2, time.Now())
```

Comment:
Can we assign a struct and add the value (2,3,2) to it so that it can be understandable. THe values seems to me like aliens. and please remove single line coments for this also.

Reply:
removed and used a struct named statuses